sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md

```diff
@@ -159,3 +183,4 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 [Getting started with SharePoint Online Management Shell](https://docs.microsoft.com/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?view=sharepoint-ps)
 [Modernize your root site](https://docs.microsoft.com/en-us/sharepoint/modern-root-site)
 [Manage site redirects](https://docs.microsoft.com/en-us/sharepoint/manage-site-redirects)
+[Page Diagnostic Tool](https://docs.microsoft.com/en-us/office365/enterprise/page-diagnostics-for-spo)
```

Comment:
"/en-us" should be removed here.

Reply:
Based on that feedback, then the  /en-us should also be removed from the other links. Did you want to make this change or do you want me to make another pull request?

Reply:
Yes, It should be removed from all of them. I will make another PR. Thanks!

Reply:
Thanks. Also note that there is also a link to the Page Diagnostic Tool further up in the article. so this should be changed as well.

Reply:
Thank you! I will make this new PR based on your feedback.